receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_scraper.go

```diff
@@ -79,7 +80,10 @@ func (s *scraper) scrape(_ context.Context) (pmetric.Metrics, error) {
 	// omit logical (virtual) filesystems (not relevant for windows)
 	partitions, err := s.partitions( /*all=*/ false)
 	if err != nil {
-		return pmetric.NewMetrics(), scrapererror.NewPartialScrapeError(err, metricsLen)
+		if len(partitions) == 0 {
+			return pmetric.NewMetrics(), scrapererror.NewPartialScrapeError(err, metricsLen)
+		}
+		s.settings.Logger.Warn("errors reporting collecting partitions information", zap.Error(err))
```

Comment:
Let's just add `err` to `errors` and proceed? Calling logger from here is not what we usually do

Reply:
ok modified.